rfcs/accessibility/accessibility.md

```diff
@@ -0,0 +1,74 @@
+
+## Background
+
+[Web Content Accessibility Guidelines (WCAG)](https://www.w3.org/TR/WCAG21/) and [WAI-ARIA Authoring Practices](https://www.w3.org/TR/wai-aria-practices-1.2/) are community driven standard for accessible web pages and applications. They provide basic framework allowing consistent usage of web pages and applications with or without assistive technologies.
+
+## Problem statement
+
+Fluent UI components aim to fully respect both WCAG and ARIA and the Fluent UI team is working with the community to improve them. From the technical point of view, thee main challenges were identified:
+
+1. Accessibility is usually not intuitive as it is not immediately obvious. It is hard to achieve high quality user experience and easy to regress it.
+2. Each screen reader and browser works differently, often making it hard or even impossible to achieve high quality user experience for all combinations. Browsers and screen readers release new versions that change the experience
+3. Not all UI/UX designs can be sufficiently represented with ARIA roles and patterns
+
+## Detailed Design or Proposal
+
+To address these challenges, following principles will be followed:
+
+- Challenge 1: Sufficient test coverage with understandable and documented tests
+- Challenge 2: Components will be created based on well tested prototypes or existing components. We will report and track issues to the screen readers vendors
+- Challenge 3: We will work with the community to improve standards. We will maintain set of best practices and validations that help consumers design accessible user interfaces
+
+### Target accessibility areas
+
+- DOM structure that provides semantical value either by using correct element types or roles
+- ARIA attributes that are valid for elements/roles used and provide correct state information about the component or element
+- Keyboard navigation (navigation by tabbing, arrow keys, pagination or letter keys, click and right click (Enter/Space, Shift+F10), and close (Escape))
+- Focus handling when the component is able to move the focus in a predictable way - mostly when opening menus, popups or dialogs or dismissing them using Esc. Focus trap for Dialog and popups
+- Sufficient contrast level
+- Light, Dark and High Contrast themes
+- Displaying focus indicator when keyboard is used to interact with them
+- Zoom up to 400%
+
+Fluent UI components will use [ability-helpers](https://github.com/microsoft/ability-helpers) for focus handling functionality, so that they can be easily integrated with application-level ability-helpers functionality such as delooser and cross-iframe focusing.
+
+### Out of scope
+
+Internationalization, globalization, keyboard shortcuts and language detection are deliberately not part of Fluent UI and should be handled by the hosting application.
```

Comment:
I just would to mention that React Spectrum handles it, for example: https://react-spectrum.adobe.com/react-spectrum/Provider.html#locales

May be at some point we will reconsider this statement :)

Reply:
maybe - currently we had props for either texts or formatting callbacks. We can consider allowing consumers to provide a dictionary, but I would like to avoid owning any particular text translations or formatting decisions as they might be dependent on different things like OS environment.